### ISSUE TITLE
[Telemetry] Update Top Site URLs

### ISSUE THREAD

**Issue by vesta0** [mozilla-mobile/fenix#15370]
- [ ] Update telemetry probes for top sites

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-7357)

**Comment by liuche** [comment]
Contact Vesta for link to the requirements, and will require talking to @mmccorks .

**Comment by liuche** [comment]
@kglazko could you summarize the conversation you had with @magorlick, specifically what request there is for Data Stewards? I don't see any documentation of the questions you need answered, but happy to answer them!

**Comment by liuche** [comment]
Update: I'll copy the existing top sites probes here, and then we can figure out what the requirements are, and what is not satisfied by the existing telemetry (and whether it's okay to collect).

**Comment by liuche** [comment]
Here are the existing top sites probes, I'll meet with @kglazko to discuss what needs to be added.

If we're looking for the specific top site being clicked (if it's a default top site) we'll need to have a clear reason for that, and it will probably need to be time-limited.

top_sites_open_frecency | event | 2021-08-01 | A user opened a frecency top site
top_sites_open_default | event | 2021-08-01 | A user opened a default top site
top_sites_open_pinned | event | 2021-08-01 | A user opened a pinned top site
top_sites_swipe_carousel | event | 2021-08-01 | A user swiped to change the page of the top sites carousel
top_sites_open_in_private_tab | A user opens a new private tab based on a top site item
top_sites_open_in_new_tab | event | 2021-08-01 | A user opens a new tab based on a top site item
top_sites_long_press
metrics_top_sites_count
metrics_has_top_sites
top_sites_remove

**Comment by eliserichards** [comment]
Talked to Vesta and got access to the doc. Next steps will be analyzing that doc and creating a proposal based on what we are allowed to track.

**Comment by eliserichards** [comment]
This is blocked on https://github.com/mozilla-mobile/android-components/issues/8491

**Comment by bifleming** [comment]
@gabrielluong is this one we can get traction on now?

**Comment by gabrielluong** [comment]
> @gabrielluong is this one we can get traction on now?

No, I think this is still blocked

**Comment by gabrielluong** [comment]
@eliserichards I don't think this is blocked anymore, but I don't know if I also understand what the requirements are either. https://github.com/mozilla-mobile/android-components/issues/8491 is mostly about removing Google default top site from certain regions now.

**Comment by bifleming** [comment]
@amedyne @st3fan should this ticket be closed?
